client/app/cloud/project/openstack/users/token/openstack-users-token.html

```diff
@@ -42,19 +42,19 @@ <h2 class="oui-modal__title" data-translate="cpou_token_title"></h2>
                 <span data-translate="cpou_token_title_information"></span>
             </p>
 
-            <table class="oui-table">
-                <thead class="oui-table__headers">
+            <table class="oui-datagrid">
```

Comment:
why haven't you used the component since it's available now?

Reply:
Because here we have a `ng-repeat` on tbody. Not possible with the component :/